languages/clojure/exercises/concept/basic-functions/.doc/introduction.md

```diff
@@ -0,0 +1,45 @@
+In Clojure, functions are your basic tools for manipulating data.
+
+Functions can be defined by using the `def` and `fn` functions together, but in practice Clojure functions are almost always defined with the `defn` macro:
+
+```clojure
+(defn new-function
+  []
+  42)
+```
+
+A Clojure function consists of 4 required parts:
+
+- The `defn` macro (or `def fn`).
+- The name of the functions being defined.
+- A vector of arguments that the function accepts.
+- The expressions which make up the function body.
+
+Functions can also contain a [docstring](https://github.com/bbatsov/clojure-style-guide#documentation) which should be written on its own line and is placed between the function name and argument vector. Docstrings are the primary way to document Clojure code.
+
+```clojure
+(defn new-function
+  "Returns the number 42"
+  []
+  42)
+```
+
+Once a function is defined, it can be invoked anywhere in the file in which is defined. (It can be imported in other files, but that's outside the scope of this lesson). 
```

Comment:
```suggestion
Once a function is defined, it can be invoked anywhere in the file in which it is defined. (It can be imported into other files, but that's outside the scope of this lesson). 
```